### ISSUE TITLE
False positive: 'try cannot be used outside of a function' error when using try in a switch statement

### ISSUE THREAD

**Issue by kython28** [DonIsaac/zlint#258]
**Version**
Commit: 81b0de9ad3c98c3560cbf4573ec6ece6456a2c4e

**Describe the bug**
False positive error when using `try` within a switch statement. The linter incorrectly reports "try cannot be used outside of a function or test block" even though the code is within the main() function.

**To Reproduce**

```zig
fn getValue(value: u8) !u8 {
    if (value == 1) return error.Test;

    return value - 1;
}

pub fn main() !void {
    loop: switch (3) {
        3 => continue :loop try getValue(3),
        2 => continue :loop try getValue(2),
        1 => {},
        else => unreachable
    }
}
```

**Expected behavior**
No error should be reported since the try is being used inside a function.

**Screenshots**

![Image](https://github.com/user-attachments/assets/7feabdb8-e931-4b05-ae9d-05c63c7ad496)
